model/secret_http.go

```diff
@@ -27,3 +27,8 @@ type SecretCreateRequest struct {
 	Extra      map[string]interface{} `json:"extra,omitempty"`
 	UserScoped bool                   `json:"user_scoped"`
 }
+
+type SecretEditRequest struct {
+	Password *string                 `json:"password,omitempty"`
```

Comment:
I don't think we want to allow password to be updated, the only thing that should be update-able should be the extra column.

Reply:
I am ok with removing it but if I remember it correctly we were discussing it on meeting and we decided to have possibility to update also password` I also did change in [docs](https://docs.google.com/document/d/1dokYxzbqIAQrG7UYK17zROy4QTSnTwGgQifMWbjQ7VU/edit#) on that meeting.

It looks we allow to update password on Auth. model as well.

Should I still remove it ?